_dprhtml/js/navigation_common.js

```diff
@@ -183,10 +183,10 @@ DPR_G.kudvala['7'] = 6;
 DPR_G.kudvala['8'] = 7;
 DPR_G.kudvala['9'] = 8;
 DPR_G.kudvala['10'] = 9;
-DPR_G.kudvala['12'] = 10;
-DPR_G.kudvala['13'] = 11;
+DPR_G.kudvala['12'] = 11;
```

Comment:
The specific problem identified in #320 was caused by the wrong index for Book 12. However, while I was here, I investigated each entry in this map and found several books to be incorrectly mapped.

Reply:
thanks for ding a through job!